bin/jobsub_submit

```diff
@@ -170,6 +171,14 @@ def main():
     parser = get_parser()
     args = parser.parse_args()
 
+    if args.version:
```

Comment:
I like this.  Depending on if we want to expand this to other commands, we might move this code into get_parser as a callback class/function, but this works well for what we expect folks to do.